### ISSUE TITLE
Something went wrong: Error: Non-whitespace before first tag. (AADE Response)

### ISSUE THREAD

**Issue by georgetomzaridis** [georgetomzaridis/aade-publicity-search-js#2]
I know the issue, that the library crash and return "Something went wrong: Error: Non-whitespace before first tag."
This is happening because AADE request return XML data but in a weird no human readable content / type. 
It worked before and stop randomly working. I am working hard to solve this issue.

Data received from AADE Response (XML maybe):
```
\x1F�\b\x00\x00\x00\x00\x00\x04\x03�U�O\x13A\x10�W6}!!��\x16Th�3��$ �\x1F���r�-\x1B�v��mK}�⏘��\x0F�hx1\x1A��4�\x06��?�����ki�Y�\x0F�����73;ͬ}c/�Q�J�\x04��(\x15�\x13�rWx���&�6\x17�3\x137\x1C��Fe�7�/B� ��\n' + 
    '��r�Z�\x15�j6���TAȚU.\x16���UK\t\x12恓��z1{�\x1B\x13S��,o��4 \x032���g\\i�]�K��K�G��\x1A7��rl%\x1BnEֶ�Z}�gny�\x1A�P�+�u�B�U���\\��TaJ��\x07a��b�˸\x1BT6XRS�����\x1D[�pS�6��\x17K��\n' +
    "�c�����\x0F1�R�45s�<=m[�n�J)$\x04��_r�AQg�\x0E���k[7��nᅵ�M|oѶ�\b=�G�+�Ud^\x17��9, �͜�\x0E����9E�\x1B=��W�E��\x19{�W�\x14�\x1Ds�\x00�\r�68�ï\r���\t2��q\x1C�1����\x02�\x04�'��@�F\x17侠h�\x1C#��?P���\x00��\x1D\x1DD� ��˘te\b4GPY\\\x06�O��:\x00\x1F��y�\ft�\x15�@\x16��\x16\x10<�:~��\x03EB�\x03�C���E��S\f�A(N�\x1DZ�@X��\r��������ط�i�j��IQ\x0F��\x01i�r\\WTr\x12Pg~ie������������%�:G�S\x1B��3\x12B�ϋ�\x03\n" +
    '��}?Q^���w�prr2>}4��ӡy��{��|H�7�y\r�!���=R����(u �U��\x0Edl����#�:�b��/���)�\x1Ar����\x1D���^<�hO�\n' +
    'g�\\N�:��:�D�N�\f3\x0B1�\x19���6��Q�j� \x1A�fB�jd(㴪L\x06\x17&\x13\\\x04� ��pE\x10P�5�>��]��F`�h��j\\�P\x00�c�y��TVc\x14�\\��<b!�\x17�X\x02���e�0��,Q�\x1AS:\x19\x7F\x16TZ�c .d\x00M�8�j)M���JXC�\x13��ul�l�5x\r��?"\x0E�\x03x$�\'\'9��K�/��[\x0Bd\x07\x00\x00
```

**Comment by georgetomzaridis** [comment]
### Issue Solved! (Please update the npm package to >1.0.6)

Turns out the AADE sometimes encode/compress their response data using gzip without any warning or cleareance. So we fix this by decode their data using gzip (gunzip).

**Comment by georgetomzaridis** [comment]
### Issue Solved! (Please update the npm package to >1.0.6)

Turns out the AADE sometimes encode/compress their response data using gzip without any warning or cleareance. So we fix this by decode their data using gzip (gunzip).
